tabs/utils/ollama.sh

```diff
@@ -0,0 +1,194 @@
+    fi
+}
+
+create_model() {
+    clear
+    list_models
+    printf "%b\n" "${YELLOW}Let's create a new model in Ollama!${RC}"
+
+    # Prompt for base model
+    printf "%b\n" "${YELLOW}Enter the base model (e.g., llama3.1):${RC}"
+    read -r base_model
+
+    # Prompt for custom model name
+    printf "%b\n" "${YELLOW}Enter a name for the new customized model:${RC}"
+    read -r custom_model_name
+
+    # Prompt for temperature setting
+    printf "%b\n" "${YELLOW}Enter the desired temperature (higher values are more creative, lower values are more coherent, e.g., 1):${RC}"
+    read -r temperature
+
+    # Prompt for system message
+    printf "%b\n" "${YELLOW}Enter the system message for the model customization (e.g., 'You are Mario from Super Mario Bros. Answer as Mario, the assistant, only.'):${RC}"
+    read -r system_message
+
+    # Create the Modelfile
+    printf "%b\n" "${YELLOW}Creating the Modelfile...${RC}"
+    cat << EOF > Modelfile
+FROM $base_model
+
+# set the temperature to $temperature
+PARAMETER temperature $temperature
+
+# set the system message
+SYSTEM """
+$system_message
+"""
+EOF
+
+    # Create the model in Ollama
+    printf "%b\n" "${YELLOW}Creating the model in Ollama...${RC}"
+    ollama create "$custom_model_name" -f Modelfile
+    printf "%b\n" "${GREEN}Model '$custom_model_name' created successfully.${RC}"
+}
+
+# Function to remove a model
+remove_model() {
+    clear
+    printf "%b\n" "${GREEN}Installed Models${RC}"
+    installed_models=$(ollama list)
+    printf "%b\n" "${installed_models}"
+
+    printf "%b\n" "${YELLOW}Please select a model to remove:${RC}"
+    printf "%b\n" "${YELLOW}Enter the name of the model you want to remove:${RC}"
+
+    read -r model_to_remove
+
+    if echo "$installed_models" | grep -q "$model_to_remove"; then
+        printf "%b\n" "${YELLOW}Removing the model: $model_to_remove...${RC}"
+        ollama rm "$model_to_remove"
+        printf "%b\n" "${GREEN}Model '$model_to_remove' has been removed.${RC}"
+    else
+        printf "%b\n" "${RED}Model '$model_to_remove' is not installed. Exiting.${RC}"
+        exit 1
+    fi
+}
+
+menu() {
+    while true; do
+        clear
+        printf "%b\n" "${YELLOW}Please select an option:${RC}"
+        printf "1) List all models\n"
+        printf "2) Show model information\n"
+        printf "3) Create a new model\n"
+        printf "4) Run a model\n"
+        printf "5) Remove a model\n"
+        printf "6) Exit\n"
+
+        printf "%b" "${YELLOW}Enter your choice (1-5): ${RC}"
+        read -r choice
+
+        case $choice in
+            1) list_models ;;
+            2) show_model_info ;;
+            3) create_model ;;
+            4) run_model ;;
+            5) remove_model;;
+            6) printf "%b\n" "${GREEN}Exiting...${RC}"; exit 0 ;;
+            *) printf "%b\n" "${RED}Invalid choice. Please try again.${RC}" ;;
+        esac
+
+        printf "%b\n" "${YELLOW}Press Enter to continue...${RC}"
+        read -r
```

Comment:
```suggestion
        read
```

https://github.com/ChrisTitusTech/linutil/pull/416#discussion_r1760768186